js/dactylo.js

```diff
@@ -265,6 +258,8 @@ window.addEventListener('DOMContentLoaded', () => {
     } else {
       setTimeout(showLesson, 500);
     }
+
+    showQuackStatus();
```

Comment:
What bug did you solve by moving this line ?

(This kills the ducky animation when a new level has been reached.)

Reply:
When we got to 5 Quacks, we saved those to local storage, do the little ducky-dance and go back down to a single quack, which de-synchronises the quacks stored in `gQuackCount` and those in `localStorage`.

What this means in practice, is that if you just won your 5th quack at (let’s say) level 4 you’ll go to level 6 with a single quack, but if you reload the page before your first lesson at level 6, you’ll “respawn” with 5 quacks, and thus need only a single win in the next two tries to go up to level 8.

Reply:
So now there’s no ducky-dance at all. Sad.

Reply:
Nevermind, I have no idea what the fuck I did, but I only way I found to replicate the bug is if I reload the page during the animation. I’ll see if I can fix that, but I may ignore that bug if I don’t find something quickly